samples/spring-security-basic-auth/pom.xml

```diff
@@ -37,6 +37,11 @@
 			<artifactId>xsuaa-spring-boot-starter</artifactId>
 			<version>${sap.cloud.security.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpclient</artifactId>
+			<scope>provided</scope>
```

Comment:
i don't understand why httpclient with scope `provided` is needed?

Reply:
yes, required beacuse of `Factory method 'springSecurityFilterChain' threw exception; nested exception is java.lang.NoClassDefFoundError: org/apache/http/client/HttpClient`

Reply:
This was the case because SpringHttpClient provided only static methods => had to introduce singleton